concepticondata/conceptlists/Tjuka-2021-192.tsv

```diff
@@ -1,193 +1,193 @@
+Tjuka-2021-192-13	13	3604	LIGHT BLUE	LIGHT BLUE	color		
+Tjuka-2021-192-14	14	3605	LIGHT GREEN	LIGHT GREEN	color		
+Tjuka-2021-192-15	15	3706	DARK YELLOW	DARK YELLOW	color		
+Tjuka-2021-192-16	16	3707	BEIGE	BEIGE	color		
+Tjuka-2021-192-17	17	3750	KHAKI	KHAKI	color		
+Tjuka-2021-192-18	18	3751	BRIGHT YELLOW	BRIGHT YELLOW	color		
+Tjuka-2021-192-19	19	3752	LILAC	LILAC	color		
+Tjuka-2021-192-20	20	3753	MAGENTA	MAGENTA	color		
+Tjuka-2021-192-21	21	3795	DARK GREEN	DARK GREEN	color		
+Tjuka-2021-192-22	22	3914	TURQUOISE	TURQUOISE	color		
+Tjuka-2021-192-23	23	40	ENVY	ENVY	emotion		
+Tjuka-2021-192-24	24	82	ANGER	ANGER	emotion		
+Tjuka-2021-192-25	25	781	FEAR (FRIGHT)	FEAR (FRIGHT)	emotion		
+Tjuka-2021-192-26	26	1000	ANXIETY	ANXIETY	emotion		
+Tjuka-2021-192-27	27	1141	PITY	PITY	emotion		
+Tjuka-2021-192-28	28	1668	GRIEF	GRIEF	emotion		
+Tjuka-2021-192-29	29	1776	SHAME	SHAME	emotion		
+Tjuka-2021-192-30	30	1783	PAIN	PAIN	emotion		
+Tjuka-2021-192-31	31	1912	RESPECT (NOUN)	RESPECT (NOUN)	emotion		
+Tjuka-2021-192-32	32	2016	JEALOUSY	JEALOUSY	emotion		
+Tjuka-2021-192-33	33	2465	DISGUST	DISGUST	emotion		
+Tjuka-2021-192-34	34	2524	JOY	JOY	emotion		
+Tjuka-2021-192-35	35	2553	MISERY	MISERY	emotion		
+Tjuka-2021-192-36	36	2941	HAPPINESS	HAPPINESS	emotion		
+Tjuka-2021-192-37	37	2960	LONGING (WISH)	LONGING (WISH)	emotion		
+Tjuka-2021-192-38	38	3049	COURAGE	COURAGE	emotion		
+Tjuka-2021-192-39	39	3211	HOPE (FAITH)	HOPE (FAITH)	emotion		
+Tjuka-2021-192-40	40	3468	KINDNESS	KINDNESS	emotion		
+Tjuka-2021-192-41	41	3673	MOURNING	MOURNING	emotion		
+Tjuka-2021-192-42	42	3749	HATE (LOATHING)	HATE (LOATHING)	emotion		
+Tjuka-2021-192-43	43	3844	CONTEMPT	CONTEMPT	emotion		
+Tjuka-2021-192-44	44	3845	DISAPPOINTMENT	DISAPPOINTMENT	emotion		
+Tjuka-2021-192-45	45	3846	EXCITEMENT	EXCITEMENT	emotion		
+Tjuka-2021-192-46	46	3847	INTEREST (FEELING)	INTEREST (FEELING)	emotion		
+Tjuka-2021-192-47	47	3848	SURPRISE (FEELING)	SURPRISE (FEELING)	emotion		
+Tjuka-2021-192-48	48	3849	TRIUMPH	TRIUMPH	emotion		
+Tjuka-2021-192-49	49	3850	AMUSEMENT	AMUSEMENT	emotion		
+Tjuka-2021-192-50	50	3851	AWE	AWE	emotion		
+Tjuka-2021-192-51	51	3853	CONFUSION	CONFUSION	emotion		
+Tjuka-2021-192-52	52	3854	SADNESS	SADNESS	emotion		
+Tjuka-2021-192-53	53	3877	DEPRESSION	DEPRESSION	emotion		
+Tjuka-2021-192-54	54	3915	PRIDE	PRIDE	emotion		
+Tjuka-2021-192-55	55	3916	PLEASURE	PLEASURE	emotion		
+Tjuka-2021-192-56	56	3834	LOVE (AFFECTION)	LOVE (AFFECTION)	emotion		
+Tjuka-2021-192-57	57	802	ADAM'S APPLE	ADAM'S APPLE	human body part		
+Tjuka-2021-192-58	58	678	BEARD	BEARD	human body part		
+Tjuka-2021-192-59	59	1402	BREAST	BREAST	human body part		
+Tjuka-2021-192-60	60	834	BUTTOCKS	BUTTOCKS	human body part		
+Tjuka-2021-192-61	61	498	CALF OF LEG	CALF OF LEG	human body part		
+Tjuka-2021-192-62	62	18	EARLOBE	EARLOBE	human body part		
+Tjuka-2021-192-63	63	1173	EYEBROW	EYEBROW	human body part		
+Tjuka-2021-192-64	64	1258	FINGERNAIL	FINGERNAIL	human body part	FINGERNAIL OR TOENAIL	1896
```

Comment:
claw or nail is also needed here for consistency.

Reply:
I didn't include CLAW OR NAIL, SKIN OR HIDE etc. because it's a mix of animal and human body parts. If we decide to add those cases as well, I'll add a few more. And there are also cases like GREEN OR UNRIPE.

Reply:
Yes, I think they should be added. Why so? If a dataset has no "green" but "green or unripe", we will say it is like "green", right? So this resolves to being a concept denoting "green". That is the idea I had here, to make sure we do not miss too much data.

Reply:
Ah, I see. I'll add the other OR concepts tomorrow.